config/database.yml

```diff
@@ -3,7 +3,9 @@ development:
   adapter: postgresql
   encoding: unicode
   database: parliamentary-questions_dev
-  host: localhost
+  username: <%= ENV['DB_USERNAME'] || ENV['PGUSER'] || 'postgres' %>
```

Comment:
We double checked and even specifying a DB_PASSWORD as long as trust based (3) option is used, the password is not even used, which is fine. The DB_USERNAME will still be needed if `postgres` doesn't exist.